build/optaplanner-build-parent/pom.xml

```diff
@@ -31,6 +31,10 @@
     <version.com.h2database>1.3.173</version.com.h2database>
     <version.com.thoughtworks.xstream>1.4.14</version.com.thoughtworks.xstream>
     <version.io.quarkus>1.10.0.Final</version.io.quarkus>
+    <!-- keep these aligned with gizmo -->
```

Comment:
Every time we upgrade Quarkus, we need to upgrade Gizmo and Jandex (it's unlikely we every upgrade gizmo or jandex indepentenly of Quarkus).
So I'd remove this comment and add one above the Quarkus version:
`<-- Keep in sync with the versions of org.quarkus.gizmo and org.jboss.jandex -->`

The Quarkus bom defines gizmo and jandex, but we only import it for optaplanner-quarkus-integration in fear of disrupting OptaPlanner's dependencies. We should set up a meeting with Radovan to see if we can and want to just import the quarkus-bom in optaplanner-build-parent (different issue). Meanwhile that comment will have to do.

@rsynek Wdyt?

Reply:
Would importing the Quarkus BOM bring any version conflicts on our existing dependencies?)

Reply:
@rsynek We should seriously consider doing that, importing the quarkus bom (not the quarkus platform bom!), but let's do that in a separate issue? Part of that issue would be researching which versions conflicts we have.